.github/workflows/scorecard.yml

```
@@ -0,0 +1,83 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
+name: Scorecard supply-chain security
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)